exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/FilterEvaluatorUtils.java

```diff
@@ -56,7 +56,8 @@ private FilterEvaluatorUtils() {
 
   public static RowsMatch evalFilter(LogicalExpression expr, MetadataBase.ParquetTableMetadataBase footer,
                                      int rowGroupIndex, OptionManager options, FragmentContext fragmentContext) {
-    List<SchemaPath> schemaPathsInExpr = new ArrayList<>(expr.accept(new FieldReferenceFinder(), null));
+    List<SchemaPath> schemaPathsInExpr = new ArrayList<>(expr.<Set<SchemaPath>,
```

Comment:
@cgivre, as discussed in mail thread, could you please add a comment which explains why this change is required with the link to JDK issue which causes a compilation error.

Also, as discussed in mail thread, please add suppression to avoid warnings in IDE.

Reply:
I'm not sure which JDK issue references this specific problem, but I put a link to the one from the email chain.